### PR TITLE
{mpi}[GCC/7.2.0-2.29] OpenMPI v3.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2470.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/33e64148de3f1f3ce011cb829352a03a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2109.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bcc5ef3c5aac60ef5da047bfbfdd56cc for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 10 out of 11 (1 easyconfigs in this PR)
nic68 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/511c50f21414a14fc36e093b48364bae for a full test report.

**Comment by boegel** [comment]
@wpoely86 

```
/usr/bin/grep: /usr/lib64/libmunge.la: No such file or directory
/usr/bin/sed: can't read /usr/lib64/libmunge.la: No such file or directory
libtool:   error: '/usr/lib64/libmunge.la' is not a valid libtool archive
```

Missing OS package on your end related to SLURM, I believe?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 10 out of 11 (1 easyconfigs in this PR)
nic68 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/38832679680e7377e127bab2422b2e95 for a full test report.

**Comment by boegel** [comment]
@wpoely86 Hmm, does this ring a bell for you?

```
../../../ompi/.libs/libmpi.so: error: undefined reference to 'ucp_request_free'
collect2: error: ld returned 1 exit status
```

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 12 out of 12 (1 easyconfigs in this PR)
ca021.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a3360fdbbdce5870fbc2f1d3ac4a3066 for a full test report.

**Comment by boegel** [comment]
@wpoely86 ping on the weird build error?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/48f80be6e779c1bad1a34e300919b432 for a full test report.

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.0.0-GCC-7.2.0-2.29.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '3.0.0'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
```
Comment (vanzod) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-3.0.0-GCC-7.2.0-2.29.eb]: Isn't this version of OpenMPI MPI-3.1 compliant?

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**